webknossos-datastore/app/com/scalableminds/webknossos/datastore/explore/ExploreLayerUtils.scala

```diff
@@ -50,8 +50,8 @@ trait ExploreLayerUtils extends FoxImplicits {
                                                    preferredVoxelSize: Option[VoxelSize],
                                                    voxelSize: VoxelSize): List[DataLayerWithMagLocators] =
     layers.map(l => {
-      val coordinateTransformations = coordinateTransformationForVoxelSize(voxelSize, preferredVoxelSize)
-      l.mapped(coordinateTransformations = coordinateTransformations)
+      val generatedCoordinateTransformation = coordinateTransformationForVoxelSize(voxelSize, preferredVoxelSize)
+      l.mapped(coordinateTransformations = generatedCoordinateTransformation.orElse(l.coordinateTransformations))
```

Comment:
I’m not sure about this orElse. Could we not have a generatedCoordinateTransformation _and_ a relevant `l.coordinateTransformations` so that we would have to combine this somehow? Or is this case not relevant?

Reply:
Lack of test datasets is of course not great :/ I don’t know of any either. Maybe it’s fair to just write a comment that this logic might not cover all cases.
However, maybe we can think about different hypothetical cases. What’s the current semantics here? the coordinateTransformationForVoxelSize is only a scale, right? When is it set? Because it looks like in that case the coordinate transformations from the metadata are not used at all? Maybe it would make sense to use the scale from the voxel size but the translation from the layer metadata? These could just be put into the matrix at the right spot, right?
I haven’t worked on this in a while so my current understanding is very rudimentary.

Reply:
We could add the translation, since the coordinateTransformations after. I am just not sure if it would work then because it's not clear to me intuitively if the translation e.g. would need to be scaled before by something.

Reply:
Ok then maybe let’s do this and write in a comment about this uncertainty. So later, when something looks off with a dataset with translation, we directly have a place to start looking.